vaadin-core/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-dev-server</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-dev-bundle</artifactId>
+        </dependency>
```

Comment:
I'm wondering: should this be added to vaadin-core or as transitive dependency to vaadin-dev-server? Otherwise people have to also exclude this dependency if they don't use it (all old users?) from their production build if they don't wanna use it / have it in use.

Reply:
Having it in `vaadin-dev-server` raises a coupling between platform and flow, since this is a platform's module.
We have a general issue about having dev mode related dependencies in production build, see https://github.com/vaadin/flow/issues/15749 . So the idea is to make `vaadin-dev-server` available asap, because the express build mode is now a default and later try to figure out the automatic mechanism for removing all that not needed for production.

Reply:
I know that issue :) Artur closed my issue about it in favor for it. I totally understand your reasoning! Wanted to mention it at least. It's easy for me and other to also exclude or from the build in the meantime.

Reply:
https://github.com/vaadin/flow/issues/15749#issuecomment-1406356660